cluster/gce/configure-vm.sh

```diff
@@ -287,6 +287,7 @@ manifest_url: '$(echo "$MANIFEST_URL" | sed -e "s/'/''/g")'
 manifest_url_header: '$(echo "$MANIFEST_URL_HEADER" | sed -e "s/'/''/g")'
 num_nodes: $(echo "${NUM_NODES}")
 e2e_storage_test_environment: '$(echo "$E2E_STORAGE_TEST_ENVIRONMENT" | sed -e "s/'/''/g")'
+kubelet_port: '$(echo "$KUBELET_PORT")'
```

Comment:
Maybe @zmerlynn can weigh in. 

My guess is that the lines not passed through sed were added recently and we just forgot to make sure they were properly sanitized. Doing a bit of digging in the git history, it looks like these were added in #16071 and #15049 in the last month.